https://redhat.atlassian.net/browse/ACM-31308 separating more files part 3 (#8884)
open-cluster-management/rhacm-docs: observability/obs_alert_forward_mcoa.adoc

.Prerequisites

> * You must have the multicluster observability add-on is enabled. For more details, see xref:../observability/obs_enable_mcoa.adoc#enable-mcoa[Enabling the multicluster observability add-on].

Reviewer: ```suggestion
* Ensure that the multicluster observability add-on is enabled. For more details, see xref:../observability/obs_enable_mcoa.adoc#enable-mcoa[Enabling the multicluster observability add-on].
```
